### ISSUE TITLE
Select treatment units is broken

### ISSUE THREAD

**Issue by arnfinn** [mong/qmongjs#209]
Using keyboard to pick unit after search is not working properly.

- when I start searching and then press Enter to pick the upper most result, a completely different unit is picked (often Ahus).
- It is not always possible to go down in your search results with arrow keys

Using the mouse to pick treatment units seems to work ok.

It is OK in commit aa19d2610f4.

I can do a `git bisect`.

Search:

![Screenshot from 2021-02-05 15-45-12](https://user-images.githubusercontent.com/136346/107048390-2e6e4c80-67c9-11eb-9bb2-de00a1bcd55a.png)

Then press Enter:
 
![Screenshot from 2021-02-05 15-45-17](https://user-images.githubusercontent.com/136346/107048403-3201d380-67c9-11eb-8ad8-db9278aea306.png)

**Comment by arnfinn** [comment]
@gabriofr Har du mulighet til å se på denne?

**Comment by arnfinn** [comment]
`git bisect` says that this is the first bad commit: c7ce42d25de8f163f3

```
Author: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Date:   Mon Jan 25 08:43:56 2021 +0000

    Bump react-select from 3.2.0 to 4.0.2 (#164)

```

**Comment by arnfinn** [comment]
Seems like it was fixed here: 15a4804916f
